docs/models/nuclear/hydrogen_tank.rst

```diff
@@ -1,37 +1,52 @@
 Hydrogen Tank
 ================
 
-The DISPATCHES Hydrogen Tank Model
+The DISPATCHES Hydrogen Tank Model represents a steady state implementation of a compressed hydrogen gas tank. This tank model supports tank filling and emptying operations for a fixed duration assuming a constant flow.
 
 Degrees of Freedom
 ------------------
 
-The Hydrogen Tank Model has X degrees of freedom.
+The Hydrogen Tank Model has 9 degrees of freedom. The degrees of freedom include the inlet state variables (flow, pressure, and temperature), previous state variables (temperature and pressure), tank dimensions (tank diameter and tank length), time duration, and outlet flow (for tank filling scenario flow = 0). By default, the tank is set to be adiabatic by fixing heat duty = 0.
 
 
 Model Structure
 ---------------
 
-The Hydrogen Tank Model consists of...
+The Hydrogen Tank Model consists of a single `ControlVolume0D` (named `control_volume`) with 2 states (`properties_in` and `properties_out`) and 2 ports (named inlet and outlet). In addition, the tank model has another state named `previous_state` that denotes the state of the tank at the beginning of time period given by `dt`. The tank model then computes holdup terms integrated over the time `dt`. Custom material and energy balance are written to account for integrated holdup terms. Finally, an internal energy balance computes the outlet temperature.
```

Comment:
Also, I think it would be important to include the custom equations, as they are fairly important for understanding how this model works and the assumptions made. We should probably also explicitly mention the assumptions we made, and why this is a quasi-steady-state model.